### PR TITLE
WCMS-1100 - adds grunt task to create a new module release

### PR DESCRIPTION
**Pull request by DaRaFF**
https://nzzdev.atlassian.net/browse/WCMS-1100

### CODE REVIEW

**Comment by DaRaFF** [comment]
@marcbachmann could you pls check it? Generally it works, but it doesn't create a tag and also doesn't push the release automatically. Do you know that behaviour on grunt-bump? or did i something wrong?

File: Gruntfile.coffee [hunk 1]
```diff
@@ -21,6 +22,39 @@ module.exports = (grunt) ->
...
+      Please run
+
+        grunt release:{{type}}
+
+      to create a release.
+      --------------------------------------------------------------------
+
+    """
+    grunt.task.run('build')
+    grunt.task.run('bump:' + type)
```
Comment (marcbachmann) [inline Gruntfile.coffee]: That should work. Sorry, won't have a detailed look at this until tomorrow ;)

**Comment by DaRaFF** [comment]
ok, it was a problem on my home machine (not creating a tag and push).

Marc already gave the ok to merge this PR.

**Comment by peyerluk** [comment]
:+1:

**Pull request merged**